dto/register-company.dto.ts

```diff
@@ -0,0 +1,29 @@
+import { IsEmail, IsNotEmpty, Length } from "class-validator";
+
+
+export class RegisterCompanyDTO {
+
+    @IsNotEmpty({ message: 'Company name is required' })

```

Comment:
Please use `IsString` something like that validation also if expected to have strings only here

Reply:
string checking is added